src/test/java/org/inferred/freebuilder/processor/MapSourceTest.java

```diff
@@ -95,14 +91,11 @@ public void test_guava_j6() {
         "",
         "  /**",
         "   * Removes the mapping for {@code key} from the map to be returned from",
-        "   * {@link Person#getName()}.",
+        "   * {@link Person#getName()}, if one is present.",
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The String literal "   * {@link Person#getName()}, if one is present." appears 4 times in this file; the first occurrence is on line 94](https://www.codacy.com/app/Chris-Purcell-39/FreeBuilder/file/2409332806/issues/source?bid=3146243&fileBranchId=3234738#l94)